app/views/comments/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<h>Comment Here</h
+<%= form_with model: @comments, url: comment_path do |form| %>
+  <%= form.text_area :text %>
+  <%= form.submit "Post Comment" %>
```

Comment:
- [ ] Clicking the button to post a comment does not work. Please try to fix this bug 🐛 
![image](https://user-images.githubusercontent.com/46839548/216740901-87647155-562f-4178-a8d7-0c39145c9916.png)

Reply:
Yes I checked and it seems there was a bug on my controller. Thank you for letting me know. 😄